lib/index.js

```diff
@@ -200,13 +199,25 @@ class PWMetrics {
 
   findMedianRun(results) {
     const ttiValues = results.map(r => r.timings.find(timing => timing.id === metrics.ids.TTI).timing);
-    const medianTTI = median(ttiValues);
+    const medianTTI = this.getMedianRun(ttiValues);
     // in the case of duplicate runs having the exact same TTI, we naively pick the first
     const medianRun = results.find(result => result.timings.find(timing => {
```

Comment:
Just committed my proposal for the code change, when you have a minute let me know if you think it's ok @paulirish